inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -128,13 +149,14 @@ def simulate_async(self,
         """Simulates the wind tunnel scenario asynchronously.
 
         Args:
+            simulator: Simulators available [OpenFOAM].
```

Comment:
Here, the same comment as Fabio's. "simulator used to run the simulation. simulators available: open foam."